mmcv/utils/registry.py

```diff
@@ -45,7 +135,27 @@ def get(self, key):
         Returns:
             class: The corresponding class.
         """
-        return self._module_dict.get(key, None)
+        scope, real_key = self.split_scope_key(key)
+        if scope is not None:
+            return self._children[scope].get(real_key)
```

Comment:
Does this satisfy the desired use case?

Reply:
Yep. The object could be built with inheritance in a recursive way.